frontend/src/components/application/OpportunityCard.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  OpportunityAssistanceListing,
+  OpportunityOverview as OpportunityOverviewType,
+} from "src/types/opportunity/opportunityResponseTypes";
+import { getConfiguredDayJs } from "src/utils/dateUtil";
+import { formatCurrency } from "src/utils/formatCurrencyUtil";
+
+import { useTranslations } from "next-intl";
+import { Grid, GridContainer, Link } from "@trussworks/react-uswds";
+
+type OpportunityOverviewProps = {
+  overview: OpportunityOverviewType;
+};
+
+const OpportunityItem = ({
+  opKey,
+  opValue,
+}: {
+  opKey: string;
+  opValue?: string | null;
+}) => {
+  return (
+    <div>
+      <dt className="margin-right-1 text-bold">{opKey}:</dt>
+      <dd>{opValue ?? "--"}</dd>
+    </div>
+  );
+};
+
+const OpportunityItemLink = ({
+  opKey,
+  opValue,
+  opId,
+}: {
+  opKey: string;
+  opValue: string | null;
+  opId: number;
+}) => {
+  return (
+    <div>
+      <dt className="margin-right-1 text-bold">{opKey}:</dt>
+      <dd>
+        <Link
+          href={`/opportunity/${opId}`}
+          className="usa-link usa-link"
+          id={`application-page-opportunity-card-link`}
+        >
+          {opValue ?? "--"}
+        </Link>
+      </dd>
+    </div>
+  );
+};
+
+const OpportunityOverview = (props: OpportunityOverviewProps) => {
+  const {
+    agency_code,
+    agency_contact_description,
+    agency_name,
+    award_ceiling,
+    award_floor,
+    close_date,
+    estimated_total_program_funding,
+    expected_number_of_awards,
+    is_cost_sharing,
+    post_date,
+    opportunity_assistance_listings,
+    opportunity_id,
+    opportunity_title,
+    opportunity_number,
+  } = props.overview;
+  const t = useTranslations("Application.opportunityOverview");
+
+  const displayOpportunityAssistanceListings = (
```

Comment:
since each of these helper functions take in their dependencies as arguments, they can function independent of the component. As such it would be a minor improvement to define them outside of the component